docker-compose.example.yml

```diff
@@ -0,0 +1,68 @@
+version: "3"
+services:
+  # The postgres server
+  database:
+    # Pulling the image in the latest tag is a disaster.
+    image: postgres:13-alpine
+    environment: 
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRE_DB=postgres
+    volumes:
+    - dbdata:/var/lib/postgresql/data
+  
+  # https://github.com/benawad/dogehouse/issues/101#issuecomment-778605325
+  adminer:
+    image: adminer
+    restart: "no"
+    ports:
+      - 5051:8080
+    depends_on:
+      - database
+  
+  # The WSS server used to establish and manage voice and text rooms
+  voice_server:
+    build:
+      context: ./shawarma
+      dockerfile: Dockerfile
+    depends_on: 
+      - backend
+
+  # The backend server
+  backend:
+    build: 
```

Comment:
Same thing, `network_mode: host` will be needed to access the PostgreSQL instance.